.github/workflows/torch-model-archiver-nightly-build.yml

```diff
@@ -1,26 +0,0 @@
-name: Push torch-model-archiver-nightly
```

Comment:
Why this workflow is  deleted?

Reply:
Merged into single workflow to save compute